test/Constraints/closures.swift

```diff
@@ -1097,3 +1097,11 @@ struct R_76250381<Result, Failure: Error> {
     return false
   }
 }
+
+// SR-13483
+(0..<10).map { x, y in } 
+// expected-error@-1 {{contextual closure type '((Range<Int>).Element) throws -> ()' expects 1 argument, but 2 were used in closure body}}
+(0..<10).map { x, y, z in } 
+// expected-error@-1 {{contextual closure type '((Range<Int>).Element) throws -> ()' expects 1 argument, but 3 were used in closure body}}
+(0..<10).map { x, y, z, w in } 
+// expected-error@-1 {{contextual closure type '((Range<Int>).Element) throws -> ()' expects 1 argument, but 4 were used in closure body}}
```

Comment:
Is there a fix-it for this situation?

Reply:
We currently don't have for those(only for all anonymous args)... I did implemented one to remove the extra arguments but reverted because in most cases e.g. `(0..<10).map { x, y in  x + y }` having a fix that remove `y` will just lead to invalid undefined variable, so I thought it would not help much.

Reply:
No worries, just checking :)

Reply:
Ah ok =]